clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -248,18 +248,26 @@ def FPAttr : CIR_Attr<"FP", "fp", [TypedAttrInterface]> {
 
 def ConstPtrAttr : CIR_Attr<"ConstPtr", "ptr", [TypedAttrInterface]> {
   let summary = "Holds a constant pointer value";
-  let parameters = (ins AttributeSelfTypeParameter<"">:$type, "uint64_t":$value);
+  let parameters = (ins AttributeSelfTypeParameter<"">:$type, "mlir::IntegerAttr":$value);
```

Comment:
Looks like it's the same change I suggested in the other PR, please incorporate this bit over there instead.